spec/models/callback_spec.rb

```diff
@@ -41,8 +41,32 @@
     end
   end
 
+  describe "#self.grouped_quotas" do
+    it "makes a success api call and returns a Hash" do
+      expect { described_class.grouped_quotas }.to_not raise_error
+      expect(described_class.grouped_quotas).to be_a(Hash)
+    end
+  end
+
   describe "#self.options" do
-    let(:options_hash) { Callback.options }
+    include_context "callback_options_hash"
     include_examples "callback_options"
   end
+
+  describe "#self.remove_current_day" do
```

Comment:
If moving the logic out to the view, this will need to move to a helper spec.

Reply:
👍